### PR TITLE
Distribute package as wheel

### PR DESCRIPTION
**Pull request by benabel**
First thank you for this excellent package, I would like to use it in [pyodide](https://pyodide.readthedocs.io/en/latest/index.html) however it needs to be packaged as a wheel. 

Sources:

- https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives

- https://packaging.python.org/key_projects/#twine

### CODE REVIEW

**Comment by BishopWolf** [comment]
[We have been waiting](https://discourse.slicer.org/t/problems-installing-lmfit-python-package/9210) for this wheel also, thank you very much for this PR.

File: 00_prepare_for_PyPI.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# This script must be run before packaging (python setup.py sdist upload).
+# This script must be run before packaging (twine upload dist/*).
```
Comment (lebigot) [inline 00_prepare_for_PyPI.sh]: Well spotted!

This instruction generally fails, though: older versions are typically present in `dist/`, and `twine` PyPI rejects them. I'll update this line.

File: setup.py [hunk 2]
```diff
@@ -385,7 +381,7 @@
     # distutils.core.setup is not like setuptools: it does not have an
     # option for automatically calling 2to3 if Python 3 is used, so the
     # conversion is done here:
-    if sys.version_info >= (3,):
+    if sys.version_info >= (3, ):
```
Comment (lebigot) [inline setup.py]: This is not standard: this is better reverted.

**Comment by lebigot** [comment]
Thanks a lot for this pull request!

I do have one question: https://github.com/lebigot/uncertainties/pull/109/files includes `if __name__ == "__main__"` at the end of `setup.py`. Do you know if this is needed?

**Comment by benabel** [comment]
@lebigot I don't think `if __name__ == "__main__"` is required as `setup.py` should always be run from python.

**Comment by lebigot** [comment]
Thanks. I'll leave it this way then.

**Pull request merged**